lms/djangoapps/instructor/views/data_access.py

```diff
@@ -77,10 +77,24 @@ def get_group_query_students(course_id, group_id):
     Asynchronously makes the subqueries for a group and then aggregates them once the subqueries are finished.
     """
     _group, queries, _relation = get_saved_queries(course_id, group_id)
-    #wait for the queries to finish before proceeding
+    # wait for the queries to finish before proceeding
     make_subqueries.apply_async(args=(course_id, group_id, queries)).get()
-    queried_students = retrieve_grouped_query(course_id, group_id)
-    return queried_students
+    # forcing evaluation here because we will be deleting stuff afterward
+    subqueries_qs = GroupedTempQueryForSubquery.objects.filter(grouped_id=group_id).distinct().values_list('query_id',
```

Comment:
Are you sure 'query_id' works? It seems like just `'query'` should be fine to get the ids of the TempQuery
https://docs.djangoproject.com/en/1.7/ref/models/querysets/#django.db.models.query.QuerySet.order_by